RegExp.make.js

```diff
@@ -37,7 +37,7 @@ RegExp.make = (function () {
     if (0x20 <= n && n <= 0x7e) {
       return String.fromCharCode(n).replace(UNSAFE_CHARS_CHARSET, '\\$&');
     }
-    var hex = n.toString(16);
+    const hex = n.toString(16);
```

Comment:
`npm install` doesn't seem to install all the dependencies for this JS project - how do I install closure compiler?